cpp/include/tensorrt_llm/batch_manager/runtimeBuffers.h

```diff
@@ -135,6 +135,12 @@ class RuntimeBuffers
 
 public:
     TensorPtr sequenceLengthsDevice;
+    bool runtimeIsChunkedContext;
+
+    std::unique_ptr<PromptTuningBuffers>& getPromptTuningBuffers()
+    {
+        return promptTuningBuffers;
+    }
```

Comment:
I actually don't understand why `promptTuningBuffers` has to be a private member. how about we put it to public along with Medusa/Lookahead etc and directly access without a get?